### PR TITLE
Release 1.1.8

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
Release 1.1.8

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: rasa/core/train.py [hunk 1]
```diff
@@ -118,7 +118,7 @@
                         file_importer,
                         train_path,
                         policy_config=policy_config,
-                        exclusion_percentage=current_run,
+                        exclusion_percentage=percentage,
```
Comment (tabergma) [inline rasa/core/train.py]: I missed this, when merging the original PR. It is part of another PR, that is not yet merged. Just wanted to fix it with the release.

**Comment by wochinge** [comment]
I'd be hyped if could include this boy here https://github.com/RasaHQ/rasa/pull/4100

**Comment by tabergma** [comment]
ok, let's wait

**Comment by wochinge** [comment]
merged and updated this branch 🙂

**Comment by wochinge** [comment]
@erohmensing Ready to merge and release. Anything else which is supposed to go in?

**Pull request merged**